src/transformers/quantizers/quantizer_awq.py

```diff
@@ -46,9 +46,6 @@ def __init__(self, quantization_config, **kwargs):
         super().__init__(quantization_config, **kwargs)
 
     def validate_environment(self, device_map, **kwargs):
-        if not torch.cuda.is_available():
-            raise RuntimeError("GPU is required to run AWQ quantized model.")
-
```

Comment:
Make sure to also to update  the condition `if device_map is None:`  below now that we also support cpu. 

We need to do a backend check for cuda and ipex but also 
- if version == ipex and device_map is not `None` or device_map != "cpu, then ask the user to make sure to move the model to cpu afterwards 
- elif device_map is None or device_map == "cpu" then ask the user to move it to cuda afterwards.